### PR TITLE
Completed prep-html-skeleton exercise -Andrew Owyang

### PR DESCRIPTION
**Pull request by Ajowyang**
Screenshot 2024-06-15 at 11 53 08 AM

### CODE REVIEW

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tags are necessary for a complete HTML Skeleton?
+
+- What type of content belongs within the `<head>` of an HTML document?
+
+- What type of content belongs within the `<body>` of an HTML document?
+
+- Where must the `DOCTYPE` declaration appear in a valid HTML document?
+
```
Comment (shawnkost) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md]: Don't forget about the quiz questions! You will need to answer these before submitting your work

**Review by shawnkost — CHANGES_REQUESTED** [review]
Code looks good Andrew! We are just missing the quiz questions

**Comment by Ajowyang** [comment]
I added the answers to the quiz questions on the md file, pushed it, and created another pull request. Is this what I'm supposed to do when you ask me to make more changes after my initial pull request?

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**